entries/c/choko.link.json

```diff
@@ -0,0 +1,15 @@
+{
+  "Choko.Link": {
+    "domain": "choko.link",
+    "img": "choko.link.png",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://support.choko.link/hc/choko/articles/1751036302-2fa",
+    "recovery": "https://support.choko.link/hc/choko/articles/1751036302-2fa",
```

Comment:
The recovery element isn't needed when it points to the same page as the documentation.
```suggestion
```